styles/signup.scss

```diff
@@ -52,4 +48,9 @@ td {
     flex: 1 1 500px;
     margin: 0px 20px;
   }
+}
+
+#bigLogo{
```

Comment:
it was explicitly requested in #22 that the logo appear in front of the animations

Reply:
I wrote #22. I never said anything about the signup page. The logo _is_ in front of the banner and always has been on the signup page. I never wanted the mix-blend mode to change, and so this is deliberately undoing parts of #43.